docs/build_process.rst

```diff
@@ -48,53 +48,81 @@ build is called the "arrangement".
+for their platform. After all worker builds finish, orchestrator build continues with its own tasks.
+
+It allows automated triggered rebuilds (where the orchestrator cluster has
+complete knowledge of the latest build configurations and their triggers).
 
-In this arrangement, the orchestrator build chooses a worker cluster
-and creates a worker build in that cluster. From then on, the worker
-build performs all the tasks required.
+In this arrangement version, environment parameters are provided by **reactor_config**.
+The order of plugins is the same, but hard coded, or placeholder, environment
+parameters in **orchestrator_inner** and **worker_inner** json files change.
 
-Delegating this way allows for future features such as multi-platform
-builds (with multiple worker builds per orchestrator build) and
-automated triggered rebuilds (where the orchestrator cluster has
-complete knowledge of the latest build configurations and their
-triggers).
+An osbs-client configuration option **reactor_config_map** is required to define
+the name of the ``ConfigMap`` object holding **reactor_config**. This
+configuration option is mandatory for arrangement versions greater than or
+equal to 6. The existing osbs-client configuration **reactor_config_secret**
+is deprecated (for all arrangements).
 
-.. graphviz:: images/arrangement-v1.dot
-   :caption: Orchestrator and worker builds (arrangement v1)
 
 Orchestrator
 ~~~~~~~~~~~~
 
 Steps performed by the orchestrator build are:
 
-- For layered images:
+- Get the parent images without pulling in order to inspect environment variables
+  (this is to allow substitution to be performed in the "release"
+  label)
 
-  * pull the parent image in order to inspect environment variables
-    (this is to allow substitution to be performed in the "release"
-    label)
+- Verify that parent images comes from a build that exists in Koji
 
-- Supply a value for the "release" label if it is missing and Koji
-  integration is enabled (this value is provided to the worker build)
+- Resolves composes, integration with `odcs`_.
+  See :ref:`yum-repositories-odcs-compose` for details.
+
+- For base images, the add_filesystem plugin runs in the orchestrator build as well as
+  the worker build. This is to create a single Koji "`image-build`_"
+  task to create filesystems for all required architectures.
+
+- Supply a value for the "release" label if it is missing
+  (this value is provided to the worker build)
+
+- Reserves build in the Koji
 
 - Apply labels supplied via build request parameter to the Dockerfile
 
 - Parse server-side configuration for atomic-reactor in order to know
   which worker clusters may be considered
 
-- Create a worker build on one of the configured clusters, and collect
+- Create a worker builds on the configured clusters, and collect
   logs and status from it
 
+- Fetches workers metadata (see :ref:`Metadata Fragment Storage`)
```

Comment:
```suggestion
- Fetch worker's metadata (see :ref:`Metadata Fragment Storage`)
```

Reply:
I'll use: Fetch workers metadata

because it will fetch metadata from multiple workers